### PR TITLE
Create a feedback channel after the user is created & add onboarding ad…

### PR DESCRIPTION
**Pull request by gcolotti**
…min as member of all created channels for the new user

### CODE REVIEW

**Comment by santiagosemhan** [comment]
Fixes #60

File: src/cloud/webhooks/userAfterSave.js [hunk 1]
```diff
@@ -26,17 +28,37 @@ const userAfterSave = async request => {
     );
   }
 
-  // Create new user chat channels
+  // Get parse role
+  const onboarding_role = await new Parse.Query(Parse.Role).equalTo('name', 'ONBOARDING_ADMIN').first();
+  // Get twilio users
+  const users = await new Twilio().client.chat.services(process.env.TWILIO_SERVICE_SID).users.list();
+  // Filter users by the desired role
+  const onboardingAdmins = users.filter(user => user.roleSid === onboarding_role.get('twilioRoleSID'));
```
Comment (santiagosemhan) [inline src/cloud/webhooks/userAfterSave.js:36]: Check first if `onboarding_role.get('twilioRoleSID')` is defined. Otherwise could lead unexpected errors.

**Comment by santiagosemhan** [comment]
@gcolotti please run prettier and eslint changes. I'll merge the PR after that.

**Comment by gcolotti** [comment]
> @gcolotti please run prettier and eslint changes. I'll merge the PR after that.

Im pushing a new commit with the prettier & eslint fixes, and the changes that Benji has mentioned in the last call (instead of getting all the users from twilio and filter 'em by role, just find the user with the role in parse)

**Pull request merged**